src/functionalTest/java/uk/gov/hmcts/reform/wataskmonitor/SpringBootFunctionalBaseTest.java

```diff
@@ -49,6 +50,8 @@
 @SpringBootTest
 @ActiveProfiles({"functional"})
 @RunWith(SpringIntegrationSerenityRunner.class)
+//@Import({CoreCaseDataApi.class,BearerTokenGenerator.class})
```

Comment:
remove commented code?

Reply:
I think its due to the master branch merge I've got some in consistencies. I corrected it now Ravi.